src/package.xml

```diff
@@ -757,6 +771,14 @@
         <members>Attribute__c-en_US</members>
         <members>Attribute__c-es</members>
         <members>Attribute__c-fr</members>
+        <members>Behavior_Involvement__c-ca</members>
```

Comment:
I think there should also be translation files for Case?

Reply:
Added.

Reply:
@xinnanlisf I think they need to be added to the package.xml too?